### PR TITLE
first solution

### PR DESCRIPTION
**Pull request by ivan-shpynda**
[DEMO LINK](https://ivan-shpynda.github.io/react_movies-list-filter/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+
+  handleQuery = (event) => {
+    this.setState({ query: event.target.value });
+  }
 
   render() {
+    const selectedMovies = moviesFromServer
+      .filter(movie => movie.title.toLowerCase()
+        .startsWith(this.state.query.toLowerCase()));
+
```
Comment (YuriiLiso) [inline src/App.js:19]:
```suggestion
    const selectedMovies = moviesFromServer
      .filter(movie => movie.title.toLowerCase()
        .startsWith(this.state.query.toLowerCase()));

```
Your code filter only by ```title```. Reread instruction.

**Review by YuriiLiso — CHANGES_REQUESTED** [review]
Good job, but need some changes.

File: src/App.js (hunk 1 above)
Comment (ivan-shpynda) [reply src/App.js:19]: well, there is some ambiguity in the instruction, but i think you are right

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+    query: '',
+  };
+
+  handleQuery = (event) => {
+    this.setState({ query: event.target.value });
+  }
 
   render() {
+    const selectedMovies = moviesFromServer
+      .filter(movie => ((`${movie.title} ${movie.description}`).toLowerCase()
```
Comment (varseniuk) [inline src/App.js:17]: clever 👍

**Review by varseniuk — APPROVED** [review]
Good job!

**Pull request closed without merging**